seasonofdocs/README.md

```diff
@@ -103,4 +103,36 @@ The Kubernetes special interest group (SIG) for command line interfaces (CLI), o
 
 - How to work with stakeholders in open source projects
 - How to document a new tool
-- Bleeding edge command line tools for Kubernetes
\ No newline at end of file
+- Bleeding edge command line tools for Kubernetes
+
+Summer of docs
+
+### Thanos: Complete Katacoda tutorials
+
+This is a great project for a writer with some passion for Docker and Kubernetes, interested in doing interactive docs: specifically learning to build [Katacoda Course](https://katacoda.com/scenario-examples/scenarios/create-course).
```

Comment:
Let's add internal issue link and who is the mentor WDYT? @povilasv

Reply:
Added in https://github.com/cncf/mentoring/pull/193/files#diff-c29d1af42af4082493795ee8f5ccbcf2R130